packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -181,7 +181,7 @@ curl -v -X GET \
 
 Returns a list of registered Inline Hooks, optionally filtered by Inline Hook type if you supply a `type` query parameter.
```

Comment:
Returns a list of registered Inline Hooks that are optionally filtered by Inline Hook type if you supply a `type` query parameter.